records_mover/db/quoting.py

```diff
@@ -1,17 +1,8 @@
 from sqlalchemy.engine import Engine, Connection
-import sqlalchemy_redshift.dialect
-from typing import Optional, Union
+from typing import Union
 
 
-def _dialect(db: Optional[Union[Connection, Engine]]):
-    if db is None:
-        # use a common one which is likely to be compatible
-        return sqlalchemy_redshift.dialect.RedshiftDialect()
-    else:
-        return db.dialect
-
-
-def quote_schema_and_table(db: Optional[Union[Connection, Engine]],
+def quote_schema_and_table(db: Union[Connection, Engine],
```

Comment:
It turns out nothing passes None into this anymore anyway, so we can drop the dependency on Redshift here.